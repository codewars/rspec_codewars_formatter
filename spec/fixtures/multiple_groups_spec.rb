RSpec.describe "Passing" do
  it "passes" do
    expect(true).to eq(true)
  end
end

RSpec.describe "Failing" do
  it "fails" do
    expect(false).to eq(true)
  end
end
