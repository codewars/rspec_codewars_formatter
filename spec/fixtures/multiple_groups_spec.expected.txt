
<DESCRIBE::>Passing

<IT::>passes

<PASSED::>Test Passed

<COMPLETEDIN::>0.8291

<COMPLETEDIN::>1.4522

<DESCRIBE::>Failing

<IT::>fails

<FAILED::>expected: true<:LF:>     got: false<:LF:><:LF:>(compared using ==)<:LF:><:LF:>Diff:<:LF:>@@ -1 +1 @@<:LF:>-true<:LF:>+false

<COMPLETEDIN::>17.6569

<COMPLETEDIN::>17.8807
